packages/nimble-blazor/NimbleBlazor/Components/NimbleTableColumnIcon.razor.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Components;
+
+namespace NimbleBlazor;
+
+public partial class NimbleTableColumnIcon<TKey> : NimbleTableColumnMapping<TKey>, IFractionalWidthColumn
+{
+    /// <summary>
+    /// The fractional/proportional width to use for this column
+    /// </summary>
+    [Parameter]
+    public double FractionalWidth { get; set; } = 1;
```

Comment:
I'm guessing that the reason why `FractionalWidth` and `MinPixelWidth` are defined here rather than in `NimbleTableColumnMapping` is that they are the most likely to deviate between the different mapping column instances even though they currently exist on both mapping column types?

Reply:
That's correct.